documentation/_templates/base.html

```diff
@@ -168,21 +168,9 @@
 
     </div>
 
-  {# Do not conflict with RTD insertion of analytics script #}
-  {% if not READTHEDOCS %}
```

Comment:
@drkane this was added by you. By adding the new cookie-consent analytics, do we need to keep it or is it save to delete it?